src/TSHelper.ts

```diff
@@ -651,7 +674,14 @@ export class TSHelper {
     }
 
     public static fixInvalidLuaIdentifier(name: string): string {
-        return name.replace(/[^a-zA-Z0-9_]/g, c => `_${c.charCodeAt(0).toString(16).toUpperCase()}`);
+        return name.replace(
+            /[^a-zA-Z0-9_]/g,
+            c =>
+                `_${c
+                    .charCodeAt(0)
+                    .toString(16)
+                    .toUpperCase()}`
```

Comment:
Why was this broken up like this?

Reply:
Prettier breaks call chains with 3 or more elements. Relevant issue: https://github.com/prettier/prettier/issues/3107.